composer.json

```diff
@@ -3,7 +3,7 @@
     "license": "MIT",
     "type": "project",
     "description": "Symfony Demo Application",
-    "minimum-stability": "stable",
+    "minimum-stability": "beta",
     "prefer-stable": true,
```

Comment:
due to this `prefer-stable` and the removal of `extra.symfony.require`, many transitive symfony dependencies are staying on 5.3

Reply:
You are right! This was an important mistake of mine. I've just reverted the removal of `extra.symfony.require` and now everything is on 5.4 beta1 and I see more deprecation messages. Thanks!